index.html

```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
     </div>
     <div>
         <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
+        
     </div>
+    <section>
         <ul>
             <li><a href="#"><img src="cat.svg" width="1" height="15" alt="cat"></a></li>
             <li><a href="#"><img src="bird.svg" width="1" height="15" alt="bird"></a></li>
             <li><a href="#"><img src="m.svg" width="1" height="15" alt="M"></a></li>
             <li><a href="#"><img src="n.png" width="1" height="15" alt="N"></a></li>
             <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
-         </ul>
-    <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+        </ul>
+        <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+    </section>
+
+        
+    <section class="section_works_container">
+       <header>
+            <img class="sec_in" src="snap.svg" alt="selfie">
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="time.svg" alt="date"></a></li>
+                </ul>
+            </div>
+       </header>
```

Comment:
Kindly, remove the `header` tag here we don't need it rather we can use either the `article` tag or just a `div`. And I see in all of the images `src` you used no existing images and in the page, it's not rendering anything please add images first then use a proper path for you all images.

https://user-images.githubusercontent.com/34578631/170739762-6fc30354-e4ac-46a5-9865-979e98dd5f7f.mov